x-pack/plugins/lens/common/constants.ts

```diff
@@ -88,4 +88,4 @@ export function getFullPath(id?: string) {
   return `/app/${PLUGIN_ID}${id ? getEditPath(id) : getBasePath()}`;
 }
 
-export const COLOR_MAPPING_OFF_BY_DEFAULT = true;
+export const COLOR_MAPPING_OFF_BY_DEFAULT = false;
```

Comment:
nit: does it make sense to still have and use this flag? Maybe we can remove it and the code that 'runs' through it?

Reply:
I prefer to do it like that, in order to be easier to disable it in case something goes wrong. We are planning to remove the intermediate step soon enough so at this point I think it makes sense to clean things up